packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1949,13 +1965,17 @@ export const DEFAULT_COMPOSITE_ICONS: {
     NoticePageJoinCallFailedDueToNoNetwork?: JSX.Element | undefined;
     NoticePageLeftCall?: JSX.Element | undefined;
     NoticePageRemovedFromCall?: JSX.Element | undefined;
-    OptionsCamera: JSX.Element;
-    OptionsMic: JSX.Element;
-    OptionsSpeaker: JSX.Element;
-    ParticipantItemMicOff: JSX.Element;
-    ParticipantItemScreenShareStart: JSX.Element;
-    VideoTileMicOff: JSX.Element;
+    NoticePageNotInvitedToRoom?: JSX.Element | undefined;
```

Comment:
Might be the api extractor, but these should just have the `?` and not have the `| undefined`

Reply:
I think it is the api extractor but at least it is consistent.